server/services/nps/getAndCreateNpsSurvey.ts

```diff
@@ -63,7 +63,7 @@ export const getAndCreateNpsSurvey = Meteor.bindEnvironment(async function getNp
 
 		await Banner.create(banner);
 	} catch (e) {
-		SystemLogger.error(e);
+		SystemLogger.error(e as Error);
```

Comment:
depends on https://github.com/RocketChat/Rocket.Chat/pull/24726

```suggestion
		SystemLogger.error(e);
```